pkg/monitoring/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["metrics.go"],
+    importpath = "kubevirt.io/containerized-data-importer/pkg/monitoring",
+    visibility = ["//visibility:public"],
+    deps = ["//pkg/common:go_default_library"],
+)
```

Comment:
Are you sure this is needed?

Reply:
yeah but it already got added here https://github.com/kubevirt/containerized-data-importer/pull/2226
one of the make targets re-adds this, not sure which one

Reply:
Rebased and this is now gone